kustomize/go.mod

```diff
@@ -12,23 +12,17 @@ require (
 	github.com/fluxcd/pkg/apis/kustomize v1.0.0
 	github.com/fluxcd/pkg/sourceignore v0.3.3
 	github.com/hashicorp/go-multierror v1.1.1
-	github.com/onsi/gomega v1.27.2
+	github.com/onsi/gomega v1.27.4
 	github.com/otiai10/copy v1.9.0
-	k8s.io/api v0.26.3
-	k8s.io/apiextensions-apiserver v0.26.3
-	k8s.io/apimachinery v0.26.3
+	k8s.io/api v0.27.0
+	k8s.io/apiextensions-apiserver v0.27.0
+	k8s.io/apimachinery v0.27.0
 	sigs.k8s.io/controller-runtime v0.14.5
-	sigs.k8s.io/kustomize/api v0.12.1
-	sigs.k8s.io/kustomize/kyaml v0.13.9
+	sigs.k8s.io/kustomize/api v0.13.2
+	sigs.k8s.io/kustomize/kyaml v0.14.1
 	sigs.k8s.io/yaml v1.3.0
 )
 
-// Pin kustomize to v4.5.7
```

Comment:
Please restore this and update the version in the comment, this is used to tell Flux users what Kustomize version we're using, as the api & kyaml versions are confusing.